### ISSUE TITLE
Cpu

### ISSUE THREAD

**Issue by whofixx** [MiCode/Xiaomi_Kernel_OpenSource#1192]
Can mi 9 snapdragon 855 overclocked to 2.94gz like 855+ by modify 855 kernel source?

**Comment by Kugumin** [comment]
No, CPU is locked (and seems to be hardware lock), GPU is overclockable

**Comment by whofixx** [comment]
How about snapdragon 625? Locked or not?

**Comment by 631250005** [comment]
Can k20pro 855+ set lower the cpu voltage?
